app/models/visitors_validation.rb

```diff
@@ -0,0 +1,47 @@
+class VisitorsValidation
+  include NonPersistedModel
+
+  MIN_LEAD_VISITOR_AGE = 18 # years
```

Comment:
This is duplicated in Prison also. You're using `Prison::MAX_VISITORS` so probably best to be consistent.